### ISSUE TITLE
make install Error 71 when installing Badgy

### ISSUE THREAD

**Issue by chayelheinsen** [arthurpalves/homebrew-formulae#1]
When attempting to install badgy via Homebrew, the following error occurs:

```bash
install: /private/tmp/badgy-20200602-14484-1o0eijb/.build/release/badgy: No such file or directory
make: *** [install] Error 71
```

Here is the full output of tapping the formulae and attempting to install badgy:

```bash
$ brew tap arthurpalves/formulae
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 4 taps (homebrew/core, homebrew/cask, homebrew/cask-fonts and homebrew/services).
==> New Formulae
austin  packr
==> Updated Formulae
...
==> Updated Casks
...

==> Tapping arthurpalves/formulae
Cloning into '/usr/local/Homebrew/Library/Taps/arthurpalves/homebrew-formulae'...
remote: Enumerating objects: 60, done.
remote: Counting objects: 100% (60/60), done.
remote: Compressing objects: 100% (33/33), done.
remote: Total 60 (delta 11), reused 57 (delta 8), pack-reused 0
Unpacking objects: 100% (60/60), done.
Tapped 2 formulae (85 files, 39.7KB).


---

 $ brew install badgy       
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
haproxy

==> Installing badgy from arthurpalves/formulae
==> Cloning https://github.com/arthurpalves/badgy.git
Updating /Users/chayelheinsen/Library/Caches/Homebrew/badgy--git
==> Checking out tag 0.1.2
HEAD is now at f592832 Merge pull request #3 from arthurpalves/feat/replace-iconset
HEAD is now at f592832 Merge pull request #3 from arthurpalves/feat/replace-iconset
==> make install prefix=/usr/local/Cellar/badgy/0.1.2
Last 15 lines from /Users/chayelheinsen/Library/Logs/Homebrew/badgy/01.make:
2020-06-02 12:18:28 -0400

make
install
prefix=/usr/local/Cellar/badgy/0.1.2

install: /private/tmp/badgy-20200602-14484-1o0eijb/.build/release/badgy: No such file or directory
make: *** [install] Error 71

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/arthurpalves/homebrew-formulae/issues
```

**Comment by arthurpalves** [comment]
Hi @chayelheinsen! Thanks for reporting this. I’ll take a look at it first thing in the morning.

**Comment by arthurpalves** [comment]
The step to build the executable is called `badgy:` on Makefile, the output of that, somehow, is:
```
make: Nothing to be done.
```

Changing the name of the step to `badgy_build:` works. I'm not able to understand why, but I've released 0.1.3 with the fix.
